scripts/updates/utils/termux_pkg_upgrade_version.sh

```diff
@@ -68,8 +68,8 @@ termux_pkg_upgrade_version() {
 
 	echo "INFO: package being updated to ${LATEST_VERSION}."
 
-	sed \
-		-e "s/^\(TERMUX_PKG_VERSION=\)\(.*\)\$/\1\"${EPOCH}${LATEST_VERSION}\"/g" \
+	sed -r \
+		-e "s/^(TERMUX_PKG_VERSION=\(?)(.*)\$/\1\"${EPOCH}${LATEST_VERSION}\"/g" \
```

Comment:
Not sure if this is needed in the case if the command in the review comment above works.

Reply:
Oh, now I get it, sorry.

Reply:
I almost forgot about this PR.